modules/gdscript/tests/scripts/parser/features/for_key_value_pair_with_type_specifiers.gd

```diff
@@ -0,0 +1,3 @@
+func test():
+	for u : String, v : String in { "key1" : "value1", "key2" : "value2", "key3" : "value3"}:
```

Comment:
```suggestion
	for u: String, v: String in { "key1": "value1", "key2": "value2", "key3": "value3" }:
```

If you are not testing the parsing, please follow the [code style](https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_styleguide.html). Otherwise, add a comment that you are testing the parser and are intentionally not following the style.